.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
you can delete this yaml in this PR

Reply:
Done with [b5610b1](https://github.com/opendilab/DI-engine/pull/452/commits/b5610b13bafa220341fb373ebe51edc3b89e6371)